app/controllers/ChooseGiveReasonItemDamagedController.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.actions._
+import forms.ChooseGiveReasonItemDamagedFormProvider
+import javax.inject.Inject
+import models.Mode
+import navigation.Navigator
+import pages.ChooseGiveReasonItemDamagedPage
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.UserAnswersService
+import views.html.ChooseGiveReasonItemDamagedView
+
+import scala.concurrent.Future
+
+class ChooseGiveReasonItemDamagedController @Inject()(
+                                       override val messagesApi: MessagesApi,
+                                       override val userAnswersService: UserAnswersService,
+                                       override val navigator: Navigator,
+                                       override val auth: AuthAction,
+                                       override val withMovement: MovementAction,
+                                       override val getData: DataRetrievalAction,
+                                       override val requireData: DataRequiredAction,
+                                       formProvider: ChooseGiveReasonItemDamagedFormProvider,
+                                       val controllerComponents: MessagesControllerComponents,
+                                       view: ChooseGiveReasonItemDamagedView
+                                     ) extends BaseNavigationController with AuthActionHelper {
+
+  def onPageLoad(ern: String, arc: String, mode: Mode): Action[AnyContent] =
```

Comment:
This needs to take in an `idx: Int` from the URL. So that you can add information for item `1`, `2`, `3` ... `n`